QPM/Providers/RemoteQPMDependencyResolver.cs

```diff
@@ -26,12 +28,27 @@ internal class RemoteQPMDependencyResolver : IDependencyResolver
         private readonly QPMApi api;
         private readonly Dictionary<RestoredDependencyPair, SharedConfig> cached = new Dictionary<RestoredDependencyPair, SharedConfig>();
         private readonly AndroidMkProvider androidMkProvider;
+        private static Linker? linker;
+
+        static RemoteQPMDependencyResolver()
+        {
+            try
+            {
+                linker = new();
+            }
+            catch (Exception e)
+            {
+                Console.Error.WriteLine($"Unable to use symlinks due to: \"{e.Message}\"\non {RuntimeInformation.OSDescription}, falling back to copy");
```

Comment:
This probably shouldn't be logged on static construction, instead on actual use. Ex: `if (linker is null) write message`

Reply:
Wouldn't that spam it unnecessarily every time a dependency is copied?